webtranslate/main.py

```diff
@@ -38,6 +40,13 @@
 bottle.TEMPLATE_PATH = ["./views/"]
 
 
+class NoLog200HandlerHandler(WSGIRequestHandler):
+    def log_message(self, format, *args):
+        # Only log if the status was not successful.
+        if not (200 <= int(args[1]) < 400):
+            log.info("%s - - [%s] %s", self.address_string(), self.log_date_time_string(), format % args)
```

Comment:
Wouldn't it be better to call the inherited method here instead?

Reply:
Ah, that was not mentioned in the commit message, but the default `log_message` uses `std.stderr.write`. This basically screws up cloud-logging big-time :P So this also fixes that it uses the normal logging infrastructure. Will add note of this.